include/RE/M/MenuTopicManager.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "RE/B/BGSSceneAction.h"
+#include "RE/B/BSFixedString.h"
+#include "RE/B/BSString.h"
+#include "RE/B/BSTEvent.h"
+#include "RE/B/BSTList.h"
+#include "RE/B/BSTSingleton.h"
+
+namespace RE
+{
+	class BGSDialogueBranch;
+	class DialoguePackage;
+	class MenuEventHandler;
+	class MenuOpenCloseEvent;
+	class TESQuest;
+	class TESTopic;
+	class TESTopicInfo;
+	struct PositionPlayerEvent;
+
+
+	class MenuTopicManager :
+		public BSTSingletonSDM<MenuTopicManager>,
+		public BSTEventSink<MenuOpenCloseEvent>,
+		public BSTEventSink<PositionPlayerEvent>
```

Comment:
Can you add offsets to these bases?

Reply:
Could you take a look at these? `BSTEventSink<PositionPlayerEvent>` seems to be at offset `08` but I can only guess the other two.

Reply:
```suggestion
		public BSTSingletonSDM<MenuTopicManager>,	// 0x10
		public BSTEventSink<MenuOpenCloseEvent>,	// 0x00
		public BSTEventSink<PositionPlayerEvent>	// 0x08
```